### DISCUSSION TITLE
I can't push to origin

### DISCUSSION THREAD

**Discussion by Seljuk0727** [community/community#137532] [New to GitHub]
![github destop不會用](https://github.com/user-attachments/assets/c979997c-0d46-4e15-afae-56c090e37aef)
![github沒上傳](https://github.com/user-attachments/assets/ca75d3ae-3cdf-4e3c-a9c0-5cf2a53c3845)
I tried to update the new version by Github Desktop and I showed me that the update was finished. Howerver, when I sign in my Github acc, I can only find the version two days ago.

**Comment by Azymack** [comment]
did you correctly add and commit your change before push?
and make sure you typed branch name correctly

**Comment by ryeryo** [comment]
@Seljuk0727 please check your working directory for changes that yet to be committed. you to add your file, then add a commit message and choose to either create a branch or merge to main/master.
